### DISCUSSION TITLE
Windows Explorer integration

### DISCUSSION THREAD

**Discussion by jglazer63** [RandomEngy/VidCoder#949] [General]
Just a suggestion to add a hook to windows explorer such that you can right click a video and click "send to VidCoder" or something like that to make it easier to add to the queue.

**Comment by RandomEngy** [comment]
You already asked for it here: https://github.com/RandomEngy/VidCoder/discussions/938

**Reply by jglazer63** [reply]
> You already asked for it here: #938

Ok, wasn't sure anyone saw it as there was no comment or indication of that.  So now the right people have seen it?

**Comment by RandomEngy** [comment]
Yes, I have seen it.
